easybuild/easyblocks/c/clang.py

```diff
@@ -166,13 +166,24 @@ def configure_step(self):
             self.llvm_obj_dir_stage3 = os.path.join(self.builddir, 'llvm.obj.3')
 
         if LooseVersion(self.version) >= LooseVersion('3.3'):
```

Comment:
add `and not build_option('strict') == run.ERROR` (requires https://github.com/hpcugent/easybuild-framework/pull/1295)